src/Propel/Generator/Builder/Om/templates/baseObjectMethodMagicCall.php

```diff
@@ -46,15 +46,18 @@ public function __<?php echo ($behaviorCallScript) ? 'parentCall' : 'call'?>($na
 
         if (0 === strpos($name, 'from')) {
             $format = substr($name, 4);
+            $inputData = $params[0];
+            $keyType = isset($params[1]) ? $params[1] : TableMap::TYPE_PHPNAME;
```

Comment:
This can look smth like `$keyType = $params[1] ?? TableMap::TYPE_PHPNAME;`

Reply:
Yes, that's better. Thank you!